easybuild/easyconfigs/p/PnetCDF/PnetCDF-1.12.1-iimpi-2020b.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'PnetCDF'
+version = "1.12.1"
```

Comment:
Version 1.12.2 is available. Please bump this easyconfig to this last release,

Reply:
@rarensu Any update on this one?